### ISSUE TITLE
Official Release

### ISSUE THREAD

**Issue by TheMrZZ** [Levertion/vscode-mcfunction#1]
Is an official release planned? Even if it's a pre-release, because for the moment I didn't manage to get this extension to work (the server index.js is never called).

Publishing it (and publishing mcfunction-langserver on npm alongside) would be really great!

**Comment by Levertion** [comment]
Sorry I missed this - not sure what went wrong!

An official release is planned eventually, and obviously it will be published to the vscode store when it is ready. I'm not sure I want to publish mcfunction-langserver on npm - it doesn't really seem to make much sense for me as it would only be used for this extension. Do you have a reason you want it pushed to npm?

**Comment by TheMrZZ** [comment]
No, I don't really have a use case for the release of mcfunction-langserver.

However, even if your extension is in development, if you publish it as a Preview, i'd be happy to test it and report bugs !
